cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/WidgetCollection/WidgetAttributesCollection/WidgetAttributeInput/AttributeMultipleValuesInput/index.tsx

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { withStyles } from '@material-ui/core';
+import { StyleRules } from '@material-ui/core/styles';
+import If from 'components/If';
+import KeyValuePairs from 'components/KeyValuePairs';
+import { COMMON_DELIMITER } from 'components/PluginJSONCreator/constants';
+import PluginInput from 'components/PluginJSONCreator/Create/Content/PluginInput';
+import * as React from 'react';
+
+export enum SupportedType {
```

Comment:
I am not sure what this type is intended for? Can you add some comments about it. I am trying to understand how this is related to multiple-values input and what `Value-Label pair` and `ID-Label pair` mean here.

Reply:
I have added comments at the top of this file. I hope that this is helpful.